data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,366 @@
+			`	"Of course, it has some... drawbacks, compared to its golden time, and don't expect the best price in a shipyard either. Nevertheless it is a fine vessel, capable of safely taking a respectable amount of cargo across large distances! So, what do you say, <last>? This is the opportunity of a lifetime, as they simply don't make ships like these anymore!"`
+			choice
+				`	"I will take this deal."`
+					goto take
+				`	"I have no interest in this deal."`
+			`	"A shame. If you decided not to help us, we will probably not meet again," they say, before leaving as discreetly as they came.`
+				decline
+			label take
+			action
+				give ship "Ancient Ladybug" "Good Omen"
+				fine 6500000
+				log "Accepted a somewhat shady deal for an old Hai ship, a Ladybug, that seems to still be in good functioning state."
+			`	They give you access to the ship and proceed to explain the details. "It is of no real significance, but surely you understand we cannot go trough official means for this, we will have to present it as a fine. Worry not, we will make it something of no consequence, like an old Korath forbidden tech you would not have known about, that we had to confiscate."`
+			`	You quickly realize they are not asking you for your opinion at all. In fact, you notice the fine has already been issued before you could say anything. "Oh, and you should not change its colors, you'll understand why later. Also make sure you go to the spaceport to get the cargo loaded onto your ship!"`
+				accept
+
+mission "Hai Legacy 2"
+	autosave
+	landing
+	name "Young but not that young"
+	description "Use the newly acquired old Ladybug to bring aid to the Unfettered Hai, as your flagship."
+	clearance
+	# deadline to make sure you don't just enjoy free rep
+	deadline 30
+	cargo "food (aid)" 121
+	source "Hai-home"
+	destination "Firelode"
+	blocked "You need space for the agreed rush cargo of food to Unfettered space on the Ancient Ladybug, and should use it as flagship, preferably as your only ship."
+	to offer
+		not "Hai Legacy 1: declined"
+	to accept
+		has "flagship model: Ancient Ladybug"
+	on enter
+		dialog
+			`Going into the spaceport one could say the cargo finds you more than you found it: a worker comes to you with crates, which are already allocated in your name for this ship. All that is left to do is to bring it to <destination> for the big promised sum of <payment>.`
+	on enter
+		dialog `As you take of from the planet you notice a message was left on the bridge, probably from the "associates". "You should use the Ladybug as your flagship and you should probably not bring any other ships. You'll understand why later."`
+			`	Given the dangers of Unfettered space, you figure it would be a good idea to familiarize yourself with the design, and maybe to modify it.`
+	on enter
+		system
+			attributes "unfettered"
+		"old reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)"
+		"reputation: Hai (Unfettered)" >?= 0
```

Comment:
I'm not sure this works with `on enter` - it's also got no stipulation for flying with only the flag ship actually included which makes the text a little incongruous. I know it says it's only a preference, but it could be presented better.

Reply:
I think I'll force it being flagship only from the mission before, previously I couldnt really bc the mission had to trigger, whereas now its a choice

Reply:
I added that branch